app/stores/AppStore.js

```diff
@@ -96,21 +109,23 @@ export default class AppStore extends Store {
     this.getTermsOfUseAcceptanceRequest.execute();
   };
 
+  _getSendLogsChoice = async () => {
+    const sendLogs = await this.getSendLogsChoiceRequest.execute().promise;
+    ipcRenderer.send('send-logs-choice', sendLogs);
+  };
+
+  _setSendLogsChoice = async ({ sendLogs }: { sendLogs: boolean }) => {
+    await this.setSendLogsChoiceRequest.execute(sendLogs);
+    await this._getSendLogsChoice();
```

Comment:
Ah I was trying to optimize this part. Have reverted it now to what I initially had in mind ;)